styles.css

```diff
@@ -133,19 +135,55 @@ div.project-item {
 	margin: 5px;
 }
 
+div.project-item p {
+	margin-right: 20%;
+}
+
 button.live, .code {
 	display: flexgrid;
 	justify-content: center;
+	flex-direction: column;
 	align-items: end;
 	background-color: #000000;
 	color: #34a39a;
 	border: 2px solid #34a39a;
 	border-radius: 15px;
-	padding-bottom: 3px;
-	margin-bottom: 5px;
+	padding-bottom: 2%;
+	margin: 0 0 2% 20%;
+}
+
+footer {
+	display: flex;
+	justify-content: center;
+	flex-direction: column;
+	background-color: #5911a4;
+	height: 50px;
+	border-radius: 15px;
+	margin-top: 23%;
+}
+
+h4 {
+	display: flex;
```

Comment:
is display: flex needed on h4? as you already have in place for parent container (.contacts)